doc/lsst.meas.extensions.shapeHSM/hsm-moments.rst

```diff
@@ -0,0 +1,4 @@
+.. py:currentmodule:: lsst.meas.extensions.shapeHSM
+
+HSM "adaptive" moments
```

Comment:
Why is this here if it's an empty file, not linked from anywhere?

Reply:
This is now populated.